docs/deploy.md

```diff
@@ -140,24 +135,38 @@ This project is in it's early stages of development and hence there are features
 
       1. Update the `gcp_project_id` portion to your GCP Project Id.
       1. Change `contents_of_service_account_json` to be the entire contents of your GCP Service Account JSON.
-</br>
```

Comment:
I think deleting this causes major unwanted changes to the markdown

Reply:
Are you sure? I did a view file and I saw the numbering to be accurate.

Reply:
Ah cool, you're right 👍